swagger.yaml

```diff
@@ -1,75 +1,94 @@
 swagger: '2.0'
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
+  version: "0.0.1"
+  title: OSU Meal Card API
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
-schemes:
-  - http
-basePath: /api/v0
-consumes:
-  - text/plain
+  url: https://github.com/osu-mist/meal-card-api
 produces:
-  - text/plain
+  - application/json
 paths:
-  /:
+  /diningbalances/{osuID}:
     get:
-      summary: Get information
-      description: Build and runtime information about the application.
+      tags:
+        - balances
+      description: "Get a single event by ID"
+      produces:
+        - application/json
       parameters:
-        - $ref: '#/parameters/pretty'
+        - name: osuID
+          in: path
+          description: "9 Digit OSU ID of the requested user"
+          required: true
+          type: integer
       responses:
         200:
-          description: Object containing information
+          description: "Successful response"
           schema:
-            $ref: '#/definitions/Info'
-
-parameters:
-  pretty:
-    name: pretty
-    in: query
-    type: boolean
-    required: false
-    description: If true, JSON response will be pretty-printed
+            $ref: "#/definitions/ResultObject"
+        404:
+          description: "Not found"
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: "Internal Server Error"
+          schema:
+            $ref: "#/definitions/Error"
 
 definitions:
-  Info:
+  ResultObject:
     properties:
-      name:
-        type: string
-        description: Name of the API.
-      time:
+      data:
+        $ref: "#/definitions/ResourceObject"
+  ResultObjects:
```

Comment:
This definition wasn't used, and I don't think it needs to be. Since a resourceObject represents one person, I can't think of when we would return multiple resourceobjects in a response (given the current design).